ConfidentialComputing/.github/pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+**PLEASE READ THIS ENTIRE MESSAGE**
+
+Hello, and thank you for your contribution! Please note that this repository is
+a read-only split of `googleapis/google-cloud-php`. As such, we are
+unable to accept pull requests to this repository.
+
+We welcome your pull request and would be happy to consider it for inclusion in
+our library if you follow these steps:
+
+* Clone the parent client library repository:
+
+```sh
+$ git clone git@github.com:googleapis/google-cloud-php.git
+```
+
+* Move your changes into the correct location in that library. Library code
+belongs in `{clientBase}/src`, and tests in `{clientBase}/tests`.
```

Comment:
```suggestion
belongs in `ConfidentialComputing/src`, and tests in `ConfidentialComputing/tests`.
```

Reply:
Oh I see maybe it gets updated during the merge.

Reply:
No, you're right! great catch, you found a bug!

Fixed in https://github.com/googleapis/google-cloud-php/pull/6085